R/qza_to_tse.R

```diff
@@ -33,13 +32,13 @@
 #' @export
 #' 
 #' @author
-#' Leo Lahti
-#' Noah de Gunst
+#' Leo Lahti and Noah de Gunst
 
 qza_to_tse <- function(features, tree, taxonomy, metadata, tmp) {
     # Input check
     if(missing(features)){
-        stop("No features have been specified. Please specify one to create a TSE from qza data.")
+        stop("No features have been specified. Please specify one to create a 
```

Comment:
should this be: "specify feature file path"?

Reply:
rephrased the sentence.